node/pkg/p2p/p2p.go

```diff
@@ -71,6 +72,16 @@ var (
 			Name: "wormhole_p2p_receive_channel_overflow",
 			Help: "Total number of p2p received messages dropped due to channel overflow",
 		}, []string{"type"})
+	p2pDrop = promauto.NewCounter(
+		prometheus.CounterOpts{
+			Name: "wormhole_p2p_drops",
+			Help: "Total number of messages that were dropped by libp2p",
+		})
+	p2pReject = promauto.NewCounter(
+		prometheus.CounterOpts{
+			Name: "wormhole_p2p_rejects",
+			Help: "Total number of messages rejected by libp2p",
+		})
```

Comment:
A more prometheus idiomatic approach would be more like:

```
wormhole_p2p_messages{type="dropped"} 4
wormhole_p2p_messages{type="rejected"} 5
wormhole_p2p_messages{type="successful"} 500
```

Labels are how you differentiate metrics in prometheus generally.